config/index.js

```diff
@@ -419,13 +419,13 @@ var conf = convict({
       ],
       env: 'SIGNIN_CONFIRMATION_SUPPORTED_CLIENTS'
     },
-    forceEmails: {
-      doc: 'If feature enabled, force sign-in confirmation for these email domains',
+    forceEmailRegex: {
+      doc: 'If feature enabled, force sign-in confirmation for any of these email regex',
```

Comment:
Might be a bit clearer if this went something like `...for email addresses matching this regex`.